src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,47 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
You should validate the 'movie' prop to make sure it's not null or undefined before accessing its properties to avoid potential errors.